test/functional/conftest.py

```diff
@@ -3,20 +3,20 @@
 from __future__ import unicode_literals
 
 import re
+from test.functional.mock_box.box import Box
```

Comment:
not sure what the linter is doing here, but it's causing you to make a bunch of changes that aren't what we want.

we want our imports to be grouped into 1) builtins, 2) 3rd party, 3) things from boxsdk (or test)

you may need to change the pylintrc file so it agrees with our style